packages/gatsby-plugin-image/src/components/gatsby-image.browser.tsx

```diff
@@ -68,6 +69,10 @@ export const GatsbyImageHydrator: FunctionComponent<GatsbyImageProps> = function
     }
     return null
   }
+  if (`class` in props) {
+    className = props.class
+    delete props.class
+  }
```

Comment:
Nit: if you extract `class` in the function param destructuring above then you don't need to use `in` here and don't need to `delete` here. The `props` var here is just a rest arg (a dubious pattern by itself since it's passed on verbatim so who knows what's being passed through).